src/Select.soy

```diff
@@ -39,7 +39,7 @@
 			{param expanded: $disabled ? false : $expanded_ /}
 			{param header kind="html"}
 				<button
-					class="{$buttonClass ? $buttonClass : ''} dropdown-select"
+					class="{$buttonClass ?: 'btn btn-default'} dropdown-select"
```

Comment:
I get it, @brunobasto. If we remove it from the JS, when the component is instantiated it will be re-rendered with the JS context removing the class already rendered by soy.